python/triton/language/core.py

```diff
@@ -778,12 +778,12 @@ def T(self):
         assert False, "Transposition must be created by the AST Visitor"
 
     @builtin
-    def to(self, dtype, bitcast=False, _builder=None):
+    def to(self, dtype, fp_downcast_rounding: str = None, bitcast=False, _builder=None):
```

Comment:
We don't seem to generate documentation at https://triton-lang.org/ for the Tensor class (seems like a bug?), but this docstring seems to be the place where we should document the rounding modes?